tools/landmark_registration_ls/landmark_registration_ls.xml

```diff
@@ -0,0 +1,33 @@
+<tool id="ip_landmark_registration_ls" name="Landmark Registration" version="0.0.1">
+    <description>using least squares</description>
+    <requirements>
+        <requirement type="package" version="1.2.1">scipy</requirement>
+        <requirement type="package" version="0.23.4">pandas</requirement>
+        <requirement type="package" version="1.15.2">numpy</requirement>
+    </requirements>
+    <command><![CDATA[ 
+         python '$__tool_directory__/landmark_registration_ls.py'
+         '$fn_pts1'
+         '$fn_pts2'
+         '$fn_tmat'
+]]></command>
+    <inputs>
+          <param name="fn_pts1" type="data" format="tabular" label="Path to tab-separated file with src points" />
```

Comment:
It's not a path. Maybe this label can be improved here

Reply:
I did so to make the description/label consistent with existing tools (eg, landmark_registration). 

I agree with you that this sounds a little bit strange. Maybe we fix such issues (for all tools) later?

Reply:
Yes that would be great!